python/graphstorm/gconstruct/transform.py

```diff
@@ -941,6 +946,12 @@ def call(self, feats):
         assert np.issubdtype(feats.dtype, np.integer) \
                 or np.issubdtype(feats.dtype, np.floating), \
                 f"The feature {self.feat_name} has to be integers or floats."
+        if self.truncate_dim is not None:
+            if isinstance(feats, np.ndarray):
+                feats = feats[:, :self.truncate_dim]
+            else:
+                assert isinstance(feats, ExtMemArrayWrapper)
```

Comment:
May we add a comment here about a converting the feats from an ExtMemArrayWrapper to in-memory array?